include/dao/dao.js

```diff
@@ -322,8 +322,9 @@ module.exports = function DAOModule(pb) {
                 .skip(offset);
 
             //apply sort order
-            if (options.orderBy) {
-                cursor.sort(options.orderBy);
+            var orderBy = options.order || options.orderBy;
+            if (orderBy) {
+                cursor.sort(orderBy);
```

Comment:
Are you guys using JSDoc to generate your documentation? I don't see a way to mark a single parameter as deprecated.